README.md

```diff
@@ -14,12 +14,12 @@ Pull down on mobile to enable page refresh.
 * Pull to refresh will only work when everything is scrolled to top
 
 ## Dependencies
-Mendix version 6.10 or up
+Mendix version 7.3 or up
```

Comment:
Widget should stay compatible with 6.10
Though the preview will only for the web modeler